proto/services/price/v1/price_service.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package services.price.v1;
+
+service PriceService {
+  rpc GetImmediateUsdPriceForBuy(GetImmediateUsdPriceForBuyRequest) returns (GetImmediateUsdPriceForBuyResponse) {}
+  rpc GetImmediateUsdPriceForSell(GetImmediateUsdPriceForSellRequest) returns (GetImmediateUsdPriceForSellResponse) {}
+  rpc GetImmediateUsdPriceForOptionBuy(GetImmediateUsdPriceForOptionBuyRequest) returns (GetImmediateUsdPriceForOptionBuyResponse) {}
+  rpc GetImmediateUsdPriceForOptionSell(GetImmediateUsdPriceForOptionSellRequest) returns (GetImmediateUsdPriceForOptionSellResponse) {}
+}
+
+message GetImmediateUsdPriceForBuyRequest {
+  float amount_in_satoshis = 1;
+}
+
+message GetImmediateUsdPriceForSellRequest {
+  float amount_in_satoshis = 1;
+}
+
+message GetImmediateUsdPriceForOptionBuyRequest {
+  float amount_in_satoshis = 1;
```

Comment:
those number should int, not float (same goes for all the float below)

Reply:
we will at some point switch to use bigint in the backend to enforce the use of int (and remove some of the weirdness of using number in javascript, plus bigint doesn't overflow). this may make sense to think about it at the dealer level also.